src/Blammo/Logging/LogSettings/Env.hs

```diff
@@ -29,6 +29,19 @@
 module Blammo.Logging.LogSettings.Env
   ( parse
   , parser
+
+  -- | Specifying defaults other than 'defaultLogSettings'
```

Comment:
I think I got this syntax wrong. Documented headers should be,

```hs
-- * Specifying defaults other than 'defaultLogSettings'
-- |
-- ...
```

Will try to remember to follow up.